librz/core/cmd_descs.yaml

```diff
@@ -590,9 +590,42 @@
         - name: project.rzdb
           type: RZ_CMD_ARG_TYPE_FILE
 - name: q
-  cname: cmd_quit
-  summary: Quit program with a return value
-  type: RZ_CMD_DESC_TYPE_OLDINPUT
+  summary: Quit rizin
+  subcommands:
+    - name: "q"
+      cname : cmd_quit
+      summary: quit rizin
+      args: []
+    - name: "q!"
+      cname: cmd_force_quit
+      summary: Force quit rizin
+      args: []
+    - name: "q!!"
+      cname: cmd_force_quit_without_history
+      summary: force quit rizin without saving history
+      args: []
+    - name: "q"
+      cname: quit_with_preference
+      summary: Quit rizin and choose to save projects and kill the process
+      args_str: "<y|n><y|n>"
+      type: RZ_CMD_DESC_TYPE_INNER
+      subcommands:
+        - name: qyy
```

Comment:
Do similar to `w1` / `w2`, etc.... Group `qyy` and `qyn` under `qy` and `qny`/`qnn` under `qn`.